lib/airbrakex/notifier.ex

```diff
@@ -21,14 +21,24 @@ defmodule Airbrakex.Notifier do
         |> add_error(error)
         |> add_context(Keyword.get(options, :context))
         |> add(:session, Keyword.get(options, :session))
-        |> add(:params, Keyword.get(options, :params))
+        |> add(:params, filter_parameters(Keyword.get(options, :params), Config.get(:airbrakex, :filter_parameters, [])))
         |> add(:environment, Keyword.get(options, :environment, %{}))
         |> Jason.encode!()
 
       post(url(), payload, @request_headers, http_options())
     end
   end
 
+  defp filter_parameters(params, filtered_keys) do
+    (params || %{})
+      |> Enum.map(&filter_parameter(&1, filtered_keys))
+      |> Enum.into(%{})
+  end
```

Comment:
`Enum.into` supporte un callback comme 3e argument pour mapper sur les valeur du enum. Suite à la suggestion plus haut pour avoir une valeur par défaut pour params, on peut aussi retirer le `|| %{}` ici.

```suggestion
  defp filter_parameters(params, filtered_keys) do
    Enum.into(params, %{}, &filter_parameters(&1, filtered_keys))
  end
```